lib/components/settings/layout/font/widgets/preview_tweet_card.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:dart_twitter_api/twitter_api.dart';
+import 'package:flutter/material.dart';
+import 'package:harpy/api/api.dart';
+import 'package:harpy/components/components.dart';
+
+class PreviewTweetCard extends StatelessWidget {
```

Comment:
- Should have a const constructor
- We need to find a way to disable the actions for this placeholder tweet card